content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -158,6 +158,41 @@ and
 Note that the information reported as Pod status depends on the current
 [ContainerState](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
 
+## Container States
+
+Once pod is assigned to a node by scheduler, kubelet starts creating container using container runtime.There are three possible states of containers, i.e Waiting, Running and Terminated. To check state of container, you can use `kubectl describe pod` command. State will be displayed for each container within that pod.
+
+* `Waiting`: This is default state of container. If container is not in running or terminated state, it is in waiting state. In this        state container is still creating and doing all required things like pulling images, applying secrets etc.
```

Comment:
Change to:

"* `Waiting`: Default state of a container. If a container is not in either Running or Terminated state, it is in waiting state. A container in Waiting state still runs its required operations, like pulling images, applying Secrets, etc.
   Along with this state, a message and reason about the state are displayed to provide more information:"

Reply:
Suggested changes are committed. Please check